src/components/icons/Pencil.tsx

```diff
@@ -0,0 +1,5 @@
+import {createSinglePathSVG} from './TEMPLATE'
+
+export const Pencil_Stroke2_Corner0_Rounded = createSinglePathSVG({
```

Comment:
Nit: name of the file is fine, since this can contain all pencil variations, but looks like the name of this icon is "pencil-line" as opposed to "pencil-wave" or just "pencil". Nit, but leaves room for others in the future.